cmd/xl-storage-format-v2.go

```diff
@@ -274,6 +276,11 @@ func (x xlMetaV2VersionHeader) matchesNotStrict(o xlMetaV2VersionHeader) bool {
 		x.Type == o.Type
 }
 
+// hasEC will return true if the version has erasure coding information.
+func (x xlMetaV2VersionHeader) hasEC() bool {
+	return x.EcM > 0 && x.EcN > 0
```

Comment:
Shouldn't `hasEC()` return true in the case of SNSD when it is recorded in xlMeta version header? It would be x.EcM = 1 and x.EcN = 0.

Reply:
@klauspost ^^

Reply:
Ah, it is clear from the method docs. Sorry, I didn't read it carefully earlier. LGTM.